File: temp_read_pdf.py
import PyPDF2
import sys

def extract_text(pdf_path):
    text = ""
    try:
        with open(pdf_path, 'rb') as file:
            reader = PyPDF2.PdfReader(file)
            for page in reader.pages:
                text += page.extract_text() + "\n"
        with open("C:/Users/BALERION/proyectos-automatizacion/MonitorMorXProONLINE/pdf_extracted.txt", "w", encoding="utf-8") as f:
            f.write(text)
        print("Success")
    except Exception as e:
        print(f"Error: {e}")

if __name__ == "__main__":
    extract_text("C:/Users/BALERION/Downloads/SÍNTESIS DE PRENSA 20 DE FEBRERO 2026- CONGRESO.pdf")

File: temp_read_pdf.js
import fs from 'fs';
import { createRequire } from 'module';
const require = createRequire(import.meta.url);
const pdfParse = require('pdf-parse/lib/pdf-parse.js');

const pdfPath = 'C:/Users/BALERION/Downloads/SÍNTESIS DE PRENSA 20 DE FEBRERO 2026- CONGRESO.pdf';
const dataBuffer = fs.readFileSync(pdfPath);

pdfParse(dataBuffer).then(data => {
    fs.writeFileSync('pdf_extracted.txt', data.text);
    console.log('Done rendering PDF');
}).catch(console.error);
